tox.ini

```diff
@@ -1,10 +1,20 @@
 [tox]
-envlist = py27, py34, py35, py36, py37, py38, pypy, pypy3.5
+envlist = 
+    py{27, 34, 35, 36, 37, 38, py, py3.5}
+    flake8
```

Comment:
flake8 environment doesn't show up on Travıs CI. You probably need to update `.travis.yml` as well.

I'd also rename it to 'lint'.